specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/informationProtection.json

```diff
@@ -0,0 +1,351 @@
+{
+    "swagger": "2.0",
```

Comment:
collapse to the main file
we still have changes that we are doing and we would like to have central control over them
this is reason for all my comments